tests/local/network.go

```diff
@@ -336,10 +338,11 @@ func (n *LocalNetwork) DeployTeleporterRegistryContracts(
 }
 
 func (n *LocalNetwork) GetSubnetsInfo() []interfaces.SubnetTestInfo {
-	return []interfaces.SubnetTestInfo{
-		*n.subnetsInfo[n.tmpnet.GetSubnet("A").SubnetID],
-		*n.subnetsInfo[n.tmpnet.GetSubnet("B").SubnetID],
+	subnetsInfo := make([]interfaces.SubnetTestInfo, 0)
```

Comment:
Performance doesn't matter much here, but when populating a new slice like this it is generally best to either:
- `make` the slice an initial size of 0 but an initial capacity large enough for all the entries to be added, and use `append` to add the elements to it (which will not require any additional allocs)
- `make` the slice with an initial size of the length of elements to be added, and the assign directly to the index within the for loop.

As is, it `append` could need to alloc a new array and copy all of the contents of the slice there on each iteration.

Reply:
It doesn't actually allocate a new array on every iteration. Slices have an underlying capacity. If you append to a full slice, golang will allocate many extra slots, which you may or may not end up using. This gives an amortized runtime cost of O(1), but it's still good practice to allocate a slice of exactly the length you need if it is known. 

https://blog.lu4p.xyz/posts/slices/